lib/modules/platform/default-scm.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { git } from '../../../test/util';
+import type { CommitFilesConfig } from '../../util/git/types';
+import { DefaultGitScm } from './default-scm';
+
+jest.mock('../../util/git');
+
+describe('modules/platform/default-scm', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('delegate branchExists to util/git', async () => {
+    git.branchExists.mockReturnValue(true);
+    await new DefaultGitScm().branchExists('branchName');
+    expect(git.branchExists).toHaveBeenCalledTimes(1);
+  });
+
+  it('delegate commitAndPush to util/git', async () => {
+    git.commitFiles.mockResolvedValueOnce('sha');
+    await new DefaultGitScm().commitAndPush({} as CommitFilesConfig);
```

Comment:
```suggestion
    await new DefaultGitScm().commitAndPush(partial<CommitFilesConfig>({}));
```
no as cast, hides runtime errors